packages/nodes-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "n8n-nodes-base",
-  "version": "0.65.0",
+  "version": "0.66.0",
```

Comment:
Version numbers do only get changed by us when we release a new version.

Reply:
OK, got it.